src/main/java/core/dao/UserDaoImpl.java

```diff
@@ -36,6 +38,17 @@ public void add(User user) {
         }
     }
     
+    @Override
+    public Optional<User> get(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            Query<User> userQuery = session.createQuery("from User u where u.id = :id", User.class);
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/04_spring/checklist/hw_03.html
> Don’t complicate get by id methods in your DAO, use session.get()

Reply:
I watched the mad guy from the materials to the lesson and followed his advice. I know it's a simple practice tasks, but anyway.